### PR TITLE
Android Predictions: callback and coroutines snippets together

### PR DESCRIPTION
**Pull request by jamesonwilliams**
**This updates the Android Predictions code snippets.**

The Core Kotlin library is in 0.1.1. Until it becomes stable, we will
*keep* the callback style code snippets for Kotlin in the documentation.

The Android Predictions content now show a "Kotlin - Callbacks" tab as
well as a "Kotlin - Coroutines (Beta)" tab.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/predictions/fragments/android/identify-text.md [hunk 1]
```diff
@@ -102,7 +117,22 @@ public void detectText(Bitmap image) {
 ```
 
 </amplify-block>
-<amplify-block name="Kotlin">
+<amplify-block name="Kotlin - Callbacks">
+
+```java
```
Comment (changxu0306) [inline docs/lib/predictions/fragments/android/identify-text.md:122]: * ```kotlin

Comment (jamesonwilliams) [reply docs/lib/predictions/fragments/android/identify-text.md:122]: Nice catch! Thanks

Comment (jamesonwilliams) [inline docs/lib/predictions/fragments/android/identify-text.md:122]:
```suggestion
```kotlin
```

**Review by changxu0306 — APPROVED** [review]
Ship it!

**Pull request merged**